test/e2e_test.go

```diff
@@ -432,15 +431,15 @@ func proxyRequestToPod(config *rest.Config, namespace, podname, scheme string, p
 		return nil, err
 	}
 	defer resp.Body.Close()
-	body, err := ioutil.ReadAll(resp.Body)
+	body, err := io.ReadAll(resp.Body)
 	if err != nil {
 		return nil, err
 	}
 	return body, nil
 }
 
 func setupForwarding(config *rest.Config, namespace, podname string, port int) (cancel func(), err error) {
-	hostIP := strings.TrimLeft(config.Host, "https://")
```

Comment:
`strings.TrimLeft` returns a slice of the string s with all leading Unicode code points contained in cutset removed.

`TrimPrefix` was intended here.